src/app/views/preferences/custom-vehicles/custom-vehicles.component.html

```diff
@@ -0,0 +1,51 @@
+<app-nav-bar title="Custom Vehicles"></app-nav-bar>
+<div>
+  <div class="vehicles__section">
+    <div class="vehicles__subtitle">
+      <b>Default Vehicles:</b>
+    </div>
+    <div *ngFor="let vehicle of defaultVehicles" class="vehicles__list-item vehicles__list-item--default" (click)="handleDefaultVehicleClick(vehicle)">
```

Comment:
I personally would like the switch button to only work if I click on it, not the text itself

Reply:
usually apps behave like this (for example Android Settings), so I did that on purpose

Reply:
Then this needs to be the same in @NiklasZinngrebe initial setup screen